quantum/keymap_extras/keymap_greek.h

```diff
@@ -0,0 +1,162 @@
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define GR_GRV  KC_GRV  // `
+#define GR_1    KC_1    // 1
+#define GR_2    KC_2    // 2
+#define GR_3    KC_3    // 3
+#define GR_4    KC_4    // 4
+#define GR_5    KC_5    // 5
+#define GR_6    KC_6    // 6
+#define GR_7    KC_7    // 7
+#define GR_8    KC_8    // 8
+#define GR_9    KC_9    // 9
+#define GR_0    KC_0    // 0
+#define GR_MINS KC_MINS // -
+#define GR_EQL  KC_EQL  // =
+// Row 2
+#define GR_SCLN KC_Q    // ;
+#define GR_WFSG KC_W    // ς
+#define GR_EPSL KC_E    // Ε
+#define GR_RHO  KC_R    // Ρ
+#define GR_TAU  KC_T    // Τ
+#define GR_UPSL KC_Y    // Υ
+#define GR_THET KC_U    // Θ
+#define GR_IOTA KC_I    // Ι
+#define GR_OMCR KC_O    // Ο
+#define GR_PI   KC_P    // Π
+#define GR_LBRC KC_LBRC // [
+#define GR_RBRC KC_RBRC // ]
+// Row 3
+#define GR_ALPH KC_A    // Α
+#define GR_SIGM KC_S    // Σ
+#define GR_DELT KC_D    // Δ
+#define GR_PHI  KC_F    // Φ
+#define GR_GAMM KC_G    // Γ
+#define GR_ETA  KC_H    // Η
+#define GR_XI   KC_J    // Ξ
+#define GR_KAPP KC_K    // Κ
+#define GR_LAMB KC_L    // Λ
+#define GR_TONS KC_SCLN // ΄ (dead)
+#define GR_QUOT KC_QUOT // '
+#define GR_BSLS KC_NUHS // (backslash)
+// Row 4
+#define GR_ZETA KC_Z    // Ζ
+#define GR_CHI  KC_X    // Χ
+#define GR_PSI  KC_C    // Ψ
+#define GR_OMEG KC_V    // Ω
+#define GR_BETA KC_B    // Β
+#define GR_NU   KC_N    // Ν
+#define GR_MU   KC_M    // Μ
+#define GR_COMM KC_COMM // ,
+#define GR_DOT  KC_DOT  // .
+#define GR_SLSH KC_SLSH // /
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ~ │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ ( │ ) │ _ │ + │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ : │ ΅ │   │   │   │   │   │   │   │   │ { │ } │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │   │   │   │   │ ¨ │ " │ | │    │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │        │   │   │   │   │   │   │   │ < │ > │ ? │          │
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define GR_TILD S(GR_GRV)  // ~
+#define GR_EXLM S(GR_1)    // !
+#define GR_AT   S(GR_2)    // @
+#define GR_HASH S(GR_3)    // #
+#define GR_DLR  S(GR_4)    // $
+#define GR_PERC S(GR_5)    // %
+#define GR_CIRC S(GR_6)    // ^
+#define GR_AMPR S(GR_7)    // &
+#define GR_ASTR S(GR_8)    // *
+#define GR_LPRN S(GR_9)    // (
+#define GR_RPRN S(GR_0)    // )
+#define GR_UNDS S(GR_MINS) // _
+#define GR_PLUS S(GR_EQL)  // +
+// Row 2
+#define GR_COLN S(GR_SCLN) // :
+#define GR_DIAL S(GR_WFSG) // ΅ (dead)
```

Comment:
This accent is called dialytika tonos. It might be a good idea to change the abbreviation so it can't be confused with the diaresis, which is called DIALytika in Greek.
```suggestion
#define GR_DTON S(GR_WFSG) // ΅ (dead)
```